uploadNeuroDB/bin/minc2jiv.pl

```diff
@@ -93,7 +94,8 @@
 my $compress= ($gzip ? "| gzip -c9 " : "") ;
 
 my %base_names_seen;
-MNI::FileUtilities::check_output_path("$TmpDir/") or exit 1;
+MNI::FileUtilities::check_output_path("$TmpDir/")
```

Comment:
wouldn't exit code 5 also work here?

Reply:
exit code 5 is specific to files provided as argument to the script which is why I did not use it here.